vllm/worker/model_runner.py

```diff
@@ -660,7 +660,10 @@ def _use_captured_graph(self, batch_size: int,
                             max_decode_seq_len: int) -> bool:
         return (self.decode_only and not self.runner.model_config.enforce_eager
                 and batch_size <= _BATCH_SIZES_TO_CAPTURE[-1]
-                and max_decode_seq_len <= self.runner.max_seq_len_to_capture)
+                # graphs are only captured to support up to max_num_seqs
+                and batch_size <= _get_graph_batch_size(
+                    self.scheduler_config.max_num_seqs) and
```

Comment:
Could we store the max as an instance field when capturing the graph in `capture_model()` (i.e. `batch_size_capture_list[-1]`) .. so that the check here is simpler/cheaper (it will replace both of these `batch_size <=` checks).